apps/linkedin-to-notion/src/contents/linkedin-notion-side-panel.tsx

```diff
@@ -11,7 +11,7 @@ import { supabase } from '~core/supabase';
 import { LinkedInNotionSidePanelContent } from '../components/LinkedInNotionSidePanel/LinkedInNotionSidePanelContent';
 
 export const config: PlasmoCSConfig = {
-  matches: ['https://www.linkedin.com/in/*'],
+  matches: ['https://www.linkedin.com/*'],
```

Comment:
LinkedIn is kind of a single page application so there's no real page changes. If we don't match on this URL we have to reload the page when we're on a profile to inject the CS